ImageClassification/ILSVRC2012/VGG/vgg_neon.py

```diff
@@ -83,10 +117,10 @@
 
 # configure callbacks
 top5 = TopKMisclassification(k=5)
-callbacks = Callbacks(model, eval_set=test, metric=top5, **args.callback_args)
+callbacks = Callbacks(model, eval_set=val, metric=top5, **args.callback_args)
 
-model.load_params(args.model_file)
-mets=model.eval(test, metric=TopKMisclassification(k=5))
+model.load_params(args.model_file, load_states=False)
+mets=model.eval(val, metric=TopKMisclassification(k=5))
```

Comment:
This vgg_neon.py seems to be only the inference path. 
Is it OK to merge your VGG changes to https://github.com/NervanaSystems/ModelZoo/pull/25? 
Or merge changes in https://github.com/NervanaSystems/ModelZoo/pull/25 to this PR is also OK.

Reply:
PR #25 has both training + inference and updated readme.